component/loki/process/internal/stages/drop_test.go

```diff
@@ -21,38 +21,28 @@ import (
 
 // Not all these are tested but are here to make sure the different types marshal without error
 var testDropRiver = `
-stage {
-	json {
+stage.json {
```

Comment:
I know, ideally I should have adjusted the indentation levels here, too 🙈.

Let me know if you think it's important, either in these tests or the docs.